lib/index.css

```diff
@@ -16,14 +16,149 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
+
+.menu-top-line, 
+.menu-middle-line, 
+.menu-bottom-line,
+.menu-left-x,
+.menu-right-x {
+  border-bottom: 3px solid #ffffff;
+  transition: all 0.5s ease;
+}
+
+.menu-left-x {
+  transform: rotate(45deg);
+  top: 7px;
+  position: relative;
+}
+
+.menu-right-x {
+  transform: rotate(-45deg);
+  bottom: 5px;
+  position: relative;
+}
+
+.slide-pane {
+  transition: all 0.2s ease-in-out;
+}
+
+.slide-pane_from_left {
+  margin: 52px auto 0 0;
+}
+
+.slide-pane__header,
+.slide-pane__close {
+  display: none;
+}
+
+.slide-pane__content {
+  padding: 24px 0;
+}
+
+.app-menu {
+  margin: 0;
+  padding: 0;
+}
+
+.app-menu li {
+  padding: 1rem 2rem;
+  cursor: pointer;
+}
+
+.app-menu li img {
+  width: 25px;
+  margin-right: 2rem;
+}
+
+.app-menu li a {
+  color: inherit;
+  text-decoration: none;
+  outline: none;
+}
+
+/* increase icon size */
+.app-menu li a .fa,
+.expand-submenu-button span .fa {
+  font-size: 2rem;
+}
+
+.app-menu span:not(.expand-menu-chevron) {
+  margin-right: 2rem;
+}
+
+.menu-item:hover *,
+.app-menu li a span *:hover {
+  color: #4c97f5;
+}
+
+.expand-menu-chevron {
+  margin-right: -2rem;
+  justify-self: flex-end;
+}
+
+.dropdown-header {
+  font-size: inherit;
+  line-height: normal;
+  color: inherit;
+  white-space: nowrap;
+}
+
+.expand-submenu-button {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin: 0rem 0rem 0rem -0.5rem;
+  background: transparent;
+  border: none;
+  outline: none;
+  padding-bottom: 1rem;
+}
+
+.sub-menu-container {
+  border-top: 1px solid #cccccc;
+  padding-top: 1rem;
+}
+
+.sub-menu-container li a {
+  padding: 1rem 1rem 1rem 2rem;
+}
+
+.sub-menu-container li a span {
+  color: inherit;
+}
+
+.app-menu-divider {
+  border-bottom: 1px solid #cccccc;
+  margin-bottom: 1.3rem;
+  padding-bottom: 2rem !important;
```

Comment:
My personal preference would be 1.0 bottom margin and padding.

Reply:
We might want to have a call to go over only padding once we're done with this round of review